Gordon360/Program.cs

```diff
@@ -54,6 +56,18 @@
 builder.Services.AddScoped<ISessionService, SessionService>();
 builder.Services.AddScoped<ServerUtils, ServerUtils>();
 builder.Services.AddHostedService<EventCacheRefreshService>();
+builder.Services.AddScoped<RecIM.IActivityService, RecIM.ActivityService>();
+builder.Services.AddScoped<RecIM.ISeriesService, RecIM.SeriesService>();
+builder.Services.AddScoped<RecIM.IMatchService, RecIM.MatchService>();
+builder.Services.AddScoped<RecIM.ITeamService, RecIM.TeamService>();
+builder.Services.AddScoped<RecIM.IParticipantService, RecIM.ParticipantService>();
+builder.Services.AddScoped<RecIM.ISportService, RecIM.SportService>();
+
+//Used to handle returning models created by HTTP Post methods containing FK constraints
+builder.Services.AddMvc().AddNewtonsoftJson(options =>
+    options.SerializerSettings.ReferenceLoopHandling
+        = Newtonsoft.Json.ReferenceLoopHandling.Ignore
+);
```

Comment:
Sorry, I meant line 26.

Reply:
I actually just removed this. This was originally an implementation needed to prevent infinite FK loops when I was returning full DB models. Then it became an issue when I was returning "created" objects. Since this is no longer an issue, I don't think there is a use for it anymore.